openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.test.tsx

```diff
@@ -199,4 +203,36 @@ describe('EntityUtils unit tests', () => {
       expect(sorter(item2, item1)).toBe(1);
     });
   });
+
+  describe('highlightSearchText method', () => {
+    it('should return the text with highlighted search text', () => {
+      const result = highlightSearchText(mockText, mockSearchText);
+
+      expect(result).toBe(mockHighlightedResult);
+    });
+
+    it('should return the original text if searchText is not found', () => {
+      const result = highlightSearchText(mockText, 'nonexistent');
+
+      expect(result).toBe(mockText);
+    });
+
+    it('should return an empty string if no text is provided', () => {
+      const result = highlightSearchText('', 'test');
+
+      expect(result).toBe('');
+    });
+
+    it('should return an empty string if no searchText is provided', () => {
```

Comment:
What if we call it with null or falsy value?

Reply:
Added test case when `highlightSearchText` called with null of falsy values.